chrome/script.browserify.js

```diff
@@ -86,15 +96,77 @@ function view() {
   ]);
 }
 
+function filterLogins(e) {
+  // remove executed search from input field
+  if (!fillOnSubmit && e.target.value.indexOf(domain) === 0) {
+    e.target.value = e.target.value.substr(domain.length);
+  }
+
+  // use fuzzy search to filter results
+  var filter = e.target.value.trim().split(/[\s\/]+/);
+  if (filter.length > 0) {
+    logins = resultLogins.slice(0);
+    filter.forEach(function(word) {
+      if (word.length > 0) {
+        var refine = [];
+        FuzzySort.go(word, logins, {allowTypo: false}).forEach(function(result) {
+          refine.push(result.target);
+        });
+        logins = refine.slice(0);
+      }
+    });
+
+    // fill login forms on submit rather than initiating a search
+    fillOnSubmit = logins.length > 0;
+  } else {
+    // reset the result list if the filter is empty
+    logins = resultLogins.slice(0);
+  }
+
+  // redraw the list
+  m.redraw();
+
+  // show / hide the filter hint
+  showFilterHint(logins && logins.length);
+}
+
+function searchKeyHandler(e) {
+  // switch to search mode if '\' is pressed and no filter text has been entered
+  if (e.code == "Backspace" && logins && logins.length > 0 && (!e.target.value.length || e.target.value == domain)) {
+    e.preventDefault();
+    logins = resultLogins = [];
```

Comment:
This is very minor thing and most probably we shouldn't do anything here, but posting in case you have some idea how to avoid the confusion. If not, don't worry about this.

I press Backspace and I am told that there are no passwords for github.com.

![backspace](https://user-images.githubusercontent.com/1177900/37606344-8ed985a8-2b95-11e8-9977-8b3b0604d0d8.gif)

Reply:
I will fix this.

Reply:
Done.